<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="simulacion_comboBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>forma1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>forma2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>forma3</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Seleccione el formato de ingreso de datos:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_simulacion_pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>UPLOAD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_simulacion_pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_graph_simulacion">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ingrese Nombre:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
